x-pack/plugins/fleet/server/collectors/register.ts

```diff
@@ -45,6 +45,16 @@ export interface FleetUsage extends Usage {
   agent_logs_panics_last_hour: AgentPanicLogsData['agent_logs_panics_last_hour'];
   agent_logs_top_errors?: string[];
   fleet_server_logs_top_errors?: string[];
+  agents_per_os: Array<{
```

Comment:
We could change `FleetUsage` to extend [AgentData](https://github.com/elastic/kibana/blob/main/x-pack/plugins/fleet/server/collectors/agent_collectors.ts#L61) so we wouldn't have to update in 2 places.

Reply:
Good point! Just did it.